### PR TITLE
Test.html added

### PR DESCRIPTION
**Pull request by ilikealotgithub1234**

### CODE REVIEW

**Comment by ilikealotgithub1234** [comment]
@rickyc

File: week1/Test.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+</head>
+<body>
+Email <input id = 'email' type = 'text'/><br>
+Password <input id = 'password' type = 'password'/><br>
+<button id='login'>Sign Up</button>
+
+<script>
```
Comment (rickyc) [inline week1/Test.html]: @ilikealotgithub1234 My first feedback would be to indent everything properly. Atom.io has an auto indentation button... maybe google for it.

Next comment the JavaScript. You'll be essentially Rubber Ducking. I spot a few errors that I think you can solve by rubber ducking / commenting your work.

When that's done and you commit the work, it'll ping me and I'll make some more comments :)

File: week1/Test.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      alert('incorrect.');
+    }
+  }
+
+    loginButton.text("Log in");
+
+    var data = [];
+    all[email] = data;
+    data[1] = email;
+    data[0] = password;
```
Comment (rickyc) [inline week1/Test.html]: Why not make data an object as well?

``` js
data = { email: email, password: password }
```

**Comment by rickyc** [comment]
Did you ever solve this?

File: week1/Test.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    data.email = email;
+    data.password = password;
+    console.log('username: ' + data.email+'  password: '+data.password);
+    console.log(all);
+  }
+
+
+
+  var loginButton = $('#login');
+  loginButton.click(storeData);
```
Comment (rickyc) [inline week1/Test.html:40]: This is very confusing as your login button is acting as the sign up. I would make the id reflect what it's actually doing.

File: week1/Test.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  Email <input id = 'email' type = 'text'/><br>
+  Password <input id = 'password' type = 'password'/><br>
+  <button id='login'>Sign Up</button>
+
+  <script>
+  var i = 0;
+  var all = {};
+  var email = $('#email').val();
+  var password = $('#password').val();
+  function storeData(){
```
Comment (rickyc) [inline week1/Test.html:15]: If you are storing data, you shouldn't also be checking the password to see if it's the right URL. I would do one thing one a time. You're doing too much.

Clicking the sign up button should just store the data as an object into an array. That's it.

File: week1/Test.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+      if(email.val() == all.data.email && password.val() == all.data.password){
+        alert('success! ');
+      }
+      else{
+        alert('incorrect.');
+        break;
+      }
+    }
+
+    loginButton.text("Log in");
```
Comment (rickyc) [inline week1/Test.html:28]: Then you need a second HTML form for the login button.

**Comment by ilikealotgithub1234** [comment]
@rickyc i am having trouble with this the console.log it returns [undefined, undefined]

**Comment by rickyc** [comment]
You'll need to push the latest code up, but undefined means the variable wasn't defined, therefore you need to trace where the variable is coming from and what you are setting it as.

**Comment by ilikealotgithub1234** [comment]
@rickyc i'm trying to figure out why

**Comment by ilikealotgithub1234** [comment]
@rickyc i declared data[0] data[1] before the console.log()

**Comment by ilikealotgithub1234** [comment]
@rickyc i think the problem is within the (property) and all.fdaSample[property]

**Comment by rickyc** [comment]
You'll need to push the code or I can't review it.

**Comment by ilikealotgithub1234** [comment]
@rickyc here

**Comment by rickyc** [comment]
Your last commit has 4 days old.

**Comment by ilikealotgithub1234** [comment]
Ok @rickyc i merged it

**Comment by rickyc** [comment]
😭 That's not what I said to do. I said push your latest code [to the branch] so I can check it. If you go on [github.com](github.com), you'll notice the code on this repo is old. You are only suppose to merge your PR when I give you the okay and the homework is finished. For now, on the `master` branch copy your `homework-1` folder somewhere outside the git repo. Then delete the folder inside the repo and commit it to `master`. Then create a new branch called `homework-1` and move the copied `homework-1` folder back into it and commit to `homework-1`. Open a new PR with that work.

**Pull request merged**